docs/manage-metadata-using-gravitino.md

```diff
@@ -736,136 +736,7 @@ In addition to the basic settings, Gravitino supports the following features:
-
-<Tabs>
-<TabItem value="bash" label="Bash">
-
-```bash
-curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
--H "Content-Type: application/json" -d '{
-  "name": "table",
-  "columns": [
-    {
-      "name": "id",
-      "type": "integer",
-      "nullable": true,
-      "comment": "Id of the user"
-    },
-    {
-      "name": "name",
-      "type": "varchar(2000)",
-      "nullable": true,
-      "comment": "Name of the user"
-    },
-    {
-      "name": "age",
-      "type": "short",
-      "nullable": true,
-      "comment": "Age of the user"
-    },
-    {
-      "name": "score",
-      "type": "double",
-      "nullable": true,
-      "comment": "Score of the user"
-    }
-  ],
-  "comment": "Create a new Table",
-  "properties": {
-    "format": "ORC"
-  },
-  "partitioning": [
-    {
-      "strategy": "identity",
-      "fieldName": ["score"]
-    }
-  ],
-  "distribution": {
-    "strategy": "hash",
-    "number": 4,
-    "funcArgs": [
-      {
-        "type": "field",
-        "fieldName": ["score"]
-      }
-    ]
-  },
-  "sortOrders": [
-    {
-      "direction": "asc",
-      "nullOrder": "NULLS_LAST",
-      "sortTerm":  {
-        "type": "field",
-        "fieldName": ["name"]
-      }
-    }
-  ]
-}' http://localhost:8090/api/metalakes/metalake/catalogs/catalog/schemas/schema/tables
-```
-
-</TabItem>
-<TabItem value="java" label="Java">
-
-```java
-tableCatalog.createTable(
-    NameIdentifier.of("metalake", "hive_catalog", "schema", "table"),
-    new ColumnDTO[] {
-    ColumnDTO.builder()
-    .withComment("Id of the user")
-    .withName("id")
-    .withDataType(Types.IntegerType.get())
-    .withNullable(true)
-    .build(),
-    ColumnDTO.builder()
-    .withComment("Name of the user")
-    .withName("name")
-    .withDataType(Types.VarCharType.of(1000))
-    .withNullable(true)
-    .build(),
-    ColumnDTO.builder()
-    .withComment("Age of the user")
-    .withName("age")
-    .withDataType(Types.ShortType.get())
-    .withNullable(true)
-    .build(),
-
-    ColumnDTO.builder()
-    .withComment("Score of the user")
-    .withName("score")
-    .withDataType(Types.DoubleType.get())
-    .withNullable(true)
-    .build(),
-    },
-    "Create a new Table",
-    tablePropertiesMap,
-    new Transform[] {
-    // Partition by id
-    Transforms.identity("score")
-    },
-    // CLUSTERED BY id
-    new DistributionDTO.Builder()
-    .withStrategy(Strategy.HASH)
-    .withNumber(4)
-    .withArgs(FieldReferenceDTO.of("id"))
-    .build(),
-    // SORTED BY name asc
-    new SortOrderDTO[] {
-    new SortOrderDTO.Builder()
-    .withDirection(SortDirection.ASCENDING)
-    .withNullOrder(NullOrdering.NULLS_LAST)
-    .withSortTerm(FieldReferenceDTO.of("name"))
-    .build()
-    }
-    );
-```
-
-</TabItem>
-</Tabs>
+For More about partition, distribution and sort order, please refer to the related [doc](table-partitioning-bucketing-sort-order.md).
```

Comment:
For more information please see the related document on [partitioning, bucketing, and sorting]